AdminOpsGuide/dbrepl.rst

```diff
@@ -1191,141 +1193,312 @@ Here is the code:
+
 On B:
 
-* Turn on replication.
-* Create a new replication instance file.
+* Source the environment file.
+* Create the database and the replication instance file.
 * Start the passive Source Server.
 * Start the Receiver Server.
 
-Example:
-
 .. parsed-literal::
-   source ./ydbenv A r1.20_x86_64 
-   ./db_create
-   ./repl_setup
-   ./originating_start A B 4001
    source ./ydbenv B r1.20_x86_64
-   ./db_create
-   ./repl_setup
-   ./replicating_start B 4001
-   ./repl_status
+
+* Create the database and the replication instance file.
+
+.. parsed-literal::
+  ./db_create
+  ./repl_setup
+
+* Start the passive Source Server.
+* Start the Receiver Server.
+
+.. parsed-literal::
+   $./replicating_start B 4001
+   Wed Jul 18 12:46:01 2018 : Initiating START of source server for secondary instance [dummy]
+   PID 18531 Receiver server is alive
+   PID 18532 Update process is alive
+   Wed Jul 18 12:46:01 2018 : /usr/local/lib/yottadb/r122/mupip replicate -receive -start -listenport=4001 -buffsize=1048576 -log=/home/B/receive.log
+   Wed Jul 18 12:46:01 2018 : %YDB-I-REPLINFO, GTM Replication Receiver Server with Pid [18531] started on replication instance [B]
+   Wed Jul 18 12:46:01 2018 : Attached to existing jnlpool with shmid = [1354006561] and semid = [1447886856]
+   Wed Jul 18 12:46:01 2018 : Created recvpool with shmid = [1354072105] and semid = [1447985163]
+   Wed Jul 18 12:46:01 2018 : Update Process started. PID 18532 [0x4864]
+   Wed Jul 18 12:46:01 2018 : Waiting for a connection...
+
+.. parsed-literal::
+   $./repl_status
+   -----------------------------------------------------------------
+   Source Server B:
+   -----------------------------------------------------------------
+   Wed Jul 18 12:46:57 2018 : Initiating CHECKHEALTH operation on source server pid [18529] for secondary instance name [dummy]
+   PID 18529 Source server is alive in PASSIVE mode
+   Wed Jul 18 12:46:57 2018 : Initiating SHOWBACKLOG operation on source server pid [18529] for secondary instance [dummy]
+   0 : backlog number of transactions written to journal pool and yet to be sent by the source server
+   0 : sequence number of last transaction written to journal pool
+   0 : sequence number of last transaction sent by source server
+   WARNING - Source Server is in passive mode, transactions are not being replicated
+   -----------------------------------------------------------------
+   Receiver Server B:
+   -----------------------------------------------------------------
+   PID 18531 Receiver server is alive
+   PID 18532 Update process is alive
+   0 : number of backlog transactions received by receiver server and yet to be processed by update process
+   0 : sequence number of last transaction received from Source Server and written to receive pool
+   0 : sequence number of last transaction processed by update process
 
 The shutdown sequence is as follows:
```

Comment:
This line comes under the "On B:" section. Does not seem the right organization. Maybe have a separate section for this.

Reply:
It's just the next part, only "A" and "B" are bolded. I can bold "The shutdown sequence is..." as well, if you want.

Reply:
Update: Bolded that sentence as well